conf/application.conf

```diff
@@ -125,3 +125,170 @@ microservice {
   }
 }
 
+chargeTypes {
```

Comment:
Is this the same reason as validation wasn't needed? Because it's this 'middle' service used with TTP which already has them being configurable? A call today would be great when you're free :)